src/irmin/irmin.mli

```diff
@@ -3483,8 +3483,12 @@ module type SYNC = sig
   (** The type for store heads. *)
   type commit
 
+  (** The type for remote status. *)
   type status = [ `Empty | `Head of commit ]
 
+  val pp_status : status Fmt.t
```

Comment:
to be consistent with the rest of the API it's probably better to expose a `val status_t` value somewhere.